include/OpenSpaceToolkit/Astrodynamics/Trajectory/Segment.hpp

```diff
@@ -105,6 +107,8 @@ class Segment
         /// @return Delta mass
         Mass computeDeltaMass() const;
 
+        Array<Maneuver> getManeuvers() const;
```

Comment:
I would call this `generateManeuvers` as we are generating them rather than it being a strict getter

Reply:
```suggestion
        Array<Maneuver> generateManeuvers() const;
```